### PR TITLE
switches conversiongen to use upstream binary

### PR DESCRIPTION
**Pull request by deads2k**
Updates the updateconversion script to use the upstream command directly.

/assign @mfojtik 
/assign @sttts 

@mfojtik I added an extra commit showing how to generate conversions for other packages too.  If you like, I can add them for the non-generated conversions you've been chasing.

### CODE REVIEW

File: hack/update-generated-deep-copies.sh [hunk 1]
```diff
@@ -1,8 +1,57 @@
...
+    github.com/openshift/origin/pkg/build/apis/build
+    github.com/openshift/origin/pkg/image/apis/image
+    github.com/openshift/origin/pkg/network/apis/network
+    github.com/openshift/origin/pkg/oauth/apis/oauth
+    github.com/openshift/origin/pkg/project/apis/project
+    github.com/openshift/origin/pkg/quota/apis/quota
+    github.com/openshift/origin/pkg/route/apis/route
+    github.com/openshift/origin/pkg/security/apis/security
+    github.com/openshift/origin/pkg/template/apis/template
+    github.com/openshift/origin/pkg/user/apis/user
```
[the remark this answers is not in the document]
Comment (sttts) [reply hack/update-generated-deep-copies.sh]: Doesn't block this PR of course.

**Comment by sttts** [comment]
Lgtm. Would like to see a follow-up removing the hard-coded package list for deepcopy. I can take a look when this has merged.

**Comment by deads2k** [comment]
etcd flake and deploymentconfig flake
/retest

**Pull request merged**